backend/routes/planForm.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import express from 'express'
+import prisma from '../utils/prismaClient'
+import { type User, InviteStatus, Role, EventType } from '@prisma/client'
+import { body } from 'express-validator'
+import { handleInputErrors } from '../modules/middleware'
+const planFormRouter = express.Router()
+const db = prisma
+
+planFormRouter.post('/',
+  body('description').isString(),
+  body('startDate').isString(),
+  body('startDate').isString(),
+  body('endDate').isString(),
+  body('EventType').isIn(Object.values(EventType)),
+  body('friends').isArray(),
```

Comment:
is this usage correct? this feels like it's creating a tonnnn of validation middlewares

FWIW I've used `express-validator` in conjunction with the `joi` package in the past and this is pretty standard in case you're looking for an alternative

Reply:
I believe that this is the right usage. Here's the doc: https://express-validator.github.io/docs/#basic-guide

Can you show me what you mean by using `express-validator` with `joi`. Also, I'm new to `express-validator`.

Reply:
try checking some examples out like this https://www.npmjs.com/package/express-joi-validation